src/overlays/actors/ovl_En_Vase/z_en_vase.c

```diff
@@ -40,5 +39,5 @@ void EnVase_Destroy(Actor* thisx, GlobalContext* globalCtx) {
 }
 
 void EnVase_Draw(Actor* thisx, GlobalContext* globalCtx) {
-    Gfx_DrawDListOpa(globalCtx, &D_06000000);
+    Gfx_DrawDListOpa(globalCtx, &objVaseMesh);
```

Comment:
not sure on calling these meshes, a mesh is only one part of the overall display list for an actor right? this is relevant for the names for the other display lists like in stalfos as well

Reply:
This is part of why I wanted to settle on naming schemes some time ago. I'm open to changing what I have here, but I feel we need some standard otherwise there will be inconsistencies.

Reply:
yea thats fair. I just think a safe bet until we have a more defined standard is to just call them dlists since thats what we refer to them as everywhere else

Reply:
Do we really need to focus on naming right now anyway?